src/FixedDataTableBufferedRows.js

```diff
@@ -142,7 +144,7 @@ var FixedDataTableBufferedRows = React.createClass({
 
       this._staticRowArray[i] =
         <FixedDataTableRow
-          key={i}
+          key={rowKeyGetter(rowIndex)}
```

Comment:
I think you're thinking of PR #85 regarding recycling column keys.

I think this is fine, and may be good if we think rowIndex is more likely to allow row recycling than array index.  We should sanity check though and ensure this doesn't cause any funkiness with image loading or anything like that.

If we want to be really conservative, we can also make the rowKeyGetter take 2 params, rowIndex and arrayIndex and have the default just return arrayIndex.